src/netex_idf/lines.rs

```diff
@@ -140,23 +140,48 @@ fn load_netex_lines(
                     .map(Element::text)
                     .ok();
                 let private_code = line.only_child("PrivateCode").map(Element::text);
-                let network_id: String =
-                    skip_error_and_log!(line
-                    .try_only_child("RepresentedByGroupRef")
-                    .and_then(|netref| netref.try_attribute_with("ref", extract_network_id)),
-                    LogLevel::Warn
-                    );
-                if !networks.contains_id(&network_id) {
-                    warn!("Failed to find network {} for line {}", network_id, id);
-                    continue;
-                }
                 let company_id: String = line
                     .try_only_child("OperatorRef")?
                     .try_attribute_with("ref", extract_company_id)?;
-                if !companies.contains_id(&company_id) {
-                    warn!("Failed to find company {} for line {}", company_id, id);
-                    continue;
-                }
+                let company = skip_error_and_log!(
+                    companies.get(&company_id).ok_or_else(|| format_err!(
+                        "Failed to find company {} for line {}",
+                        company_id,
+                        id
+                    )),
+                    LogLevel::Warn
+                );
+                let network_id: String = line
+                    .try_only_child("RepresentedByGroupRef")
+                    .and_then(|netref| {
+                        netref.try_attribute_with::<_, _, String>("ref", extract_network_id)
+                    })
+                    .and_then(|network_id| {
+                        networks.get(&network_id).ok_or_else(|| {
+                            format_err!("Failed to find network {} for line {}", network_id, id)
+                        })
+                    })
+                    .map(|network| network.id.clone())
+                    .or_else::<failure::Error, _>(|error| {
+                        warn!(
+                            "{}. Creating a Network from Operator '{}'.",
+                            error, company_id
+                        );
+
+                        let network_id = format!("Operator_{}", company.id);
```

Comment:
Fails and crashes if the Network (from Operator) has already been created (by a previous Line with the same Operator).
Check this existence before creating it and push it.
```
WARN Failed to find a child 'RepresentedByGroupRef' in element 'Line'. Creating a Network from Operator '100'.
WARN Failed to find a child 'RepresentedByGroupRef' in element 'Line'. Creating a Network from Operator '100'.
identifier Operator_100 already exists
```

Reply:
Done in 0b5eb84de652488ad5b9aab6ba674373dbfb3f31. I also added a unit test for that use case.